mmv1/third_party/terraform/tests/resource_network_security_tls_inspection_policy_test.go.erb

```diff
@@ -104,11 +104,25 @@ resource "google_privateca_certificate_authority" "default" {
   }
 }
 
+resource "google_project_service_identity" "ns_sa" {
+  provider = google-beta
```

Comment:
Need to add `google-beta` to other resources in this test. `TestAccNetworkSecurityTlsInspectionPolicy_update` failed possibly due to this.

Reply:
Ah- yeah, missed that it wasn't already. Those aren't needed since this is a beta test.

Reply:
I think `google_project_service_identity` is beta-only resource though https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/project_service_identity

Reply:
The google-beta block is only needed in examples! This file is beta-guarded so this should work.

Reply:
ah, got it! Sorry, didn't realized it was a beta test.